app/views/content/landing/careers/_content.html.erb

```diff
@@ -39,7 +39,7 @@
       </p>
 
       <ul>
-      <li>most teachers get a pay rise every year, depending on your school. You could earn over <%= v :salaries_fiveyears_maxshortened %> within 5 years</li>
+      <li>most teachers get a pay rise every year, depending on your school. You could earn over <%= v :salaries_fiveyears_maxshortened %> after 5 years</li>
```

Comment:
Figures all look fine, but I think the content looks strange to have two sentences on the bullet point. Normally either all bullet points should be sentences and be capitalised, or not at all. 

```suggestion
      <li>most teachers get a pay rise every year (depending on your school), so you could earn over <%= v :salaries_fiveyears_maxshortened %> after 5 years</li>
```